test/built-ins/TypedArrays/from/mapfn-this-without-thisarg-non-strict.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-%typedarray%.from
+description: >
+  Assert mapfn `this` without thisArg
+info: >
+  22.2.2.1 %TypedArray%.from ( source [ , mapfn [ , thisArg ] ] )
+
+  ...
+  5. If thisArg was supplied, let T be thisArg; else let T be undefined.
+  ...
+  10. Repeat, while k < len
+    ...
+    c. If mapping is true, then
+      i. Let mappedValue be ? Call(mapfn, T, « kValue, k »).
+  ...
+includes: [testTypedArray.js]
+flags: [noStrict]
+---*/
+
+var source = [42, 43];
+var global = (function() {
```

Comment:
It's probably best to use [the `fnGlobalObject` helper function](https://github.com/tc39/test262/blob/d549225f9151f1ea46bfcbc0b7b1447875da6b50/harness/fnGlobalObject.js) for this